src/edu/csus/ecs/pc2/clics/API202306/CLICSContestAccess.java

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+
+import javax.ws.rs.core.SecurityContext;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Contest State
+ *
+ * @author John Buck
+ *
+ */
+public class CLICSContestAccess {
+
+    // capabilities for API 2023-06
+    public static final String API_CAPABILITY_CONTEST_START = "contest_start";
+    public static final String API_CAPABILITY_CONTEST_THAW = "contest_thaw";
+    public static final String API_CAPABILITY_TEAM_SUBMIT = "team_submit";
+    public static final String API_CAPABILITY_TEAM_CLAR = "team_clar";
+    public static final String API_CAPABILITY_PROXY_SUBMIT = "proxy_submit";
+    public static final String API_CAPABILITY_PROXY_CLAR = "proxy_clar";
+    public static final String API_CAPABILITY_ADMIN_SUBMIT = "admin_submit";
+    public static final String API_CAPABILITY_ADMIN_CLAR = "admin_clar";
+    // tentative - not official, but mentioned in the spec
+    public static final String API_CAPABILITY_AWARDS_UPDATE = "awards_update";
+
+    public static final String ENDPOINT_ACCESS_METHOD = "getEndpointProperties";
+
+    @JsonProperty
+    private String [] capabilities;
+
+    @JsonProperty
+    private CLICSEndpoint [] endpoints;
+
+    IInternalController controller = null;
+
+    /**
+     * Fill in properties for contest state as per 2023-06 spec
```

Comment:
This constructor header comment references "contest state" -- but I'm not convinced this class is at all related to the CLICS notion of "contest state" (for example, as it relates to the CLICS endpoint `/contests/<id>/state`).  I think maybe the comment is supposed to say that the constructor fills in "access properties" (or something similar)?

Reply:
Yes, this was a copy / paste error from the state class.  Fixed in next push.